pkg/client/group_mgmt_client.go

```diff
@@ -84,6 +88,20 @@ func (client *GroupMgmtClient) EnableDebug() {
 	client.Client.SetDebug(true)
 }
 
+// refreshSessionToken : refresh session token
+func (client *GroupMgmtClient) refreshSessionToken(code int, message string) (bool, error) {
+	if code >= 400 && client.SessionToken != "" {
+		if strings.Contains(message, "SM_http_unauthorized") {
+			newSessionToken, err := client.login(client.Username, client.Password)
```

Comment:
Before this line, add:

client.SessionToken = ""

Otherwise a changed password can still cause infinite recursion.

Reply:
done